lib/config/index.js

```diff
@@ -129,16 +128,6 @@ function checkLimitsConfig (cfg) {
   if (!(cfg.limits.requestsBufferWarningThreshold >= cfg.limits.concurrentRequests && cfg.limits.requestsBufferWarningThreshold <= cfg.limits.requestsBufferSize)) {
     throw new KuzzleInternalError('Invalid configuration: limits.requestsBufferWarningThreshold should be comprised between limits.concurrentRequests and limits.requestsBufferSize');
   }
-
-  /*
-   Number of documents fetched or written by a single API request
-   Cannot exceed 9999 or 80% of requestsBufferSize
-   */
-  ['documentsWriteCount', 'documentsFetchCount'].forEach(prop => {
-    if (cfg.limits[prop] > 9999 || cfg.limits[prop] > .8 * cfg.limits.requestsBufferSize) {
-      throw new KuzzleInternalError(`Invalid limits.${prop} configuration: cannot exceed 80% of limits.requestsBufferSize and cannot be higher than 9999`);
-    }
-  });
 }
 
 module.exports = loadConfig();
```

Comment:
The removed code was just checking if the config number were correct according to another config.

ChatGPT was more accurate
```
In summary, this code snippet is used to validate the configuration related to the number of documents fetched or written by an API request. It ensures that these values do not exceed certain limits (9999 or 80% of the request buffer size) and raises an error if the configuration violates these constraints. This kind of validation is common in software development to prevent unexpected behavior or issues due to misconfigured parameters.
```